### DISCUSSION TITLE
Duplicate a Task in another Project

### DISCUSSION THREAD

**Discussion by LBAK420** [kanboard/kanboard#5438] [General]
I defined a Destination Column in the class TaskDuplicateAnotherProject  and the target is that I wanna duplicate a task from a specific project to an another specific Project of Choice and in a specific Column of the choice Project.
The Problem is that in the Destination Column it shows the Standard Columns not Columns of the Choice Projekt that the Task has to be duplicated. Any Advice? tnx 
The path to the Class: "C:\xampp\htdocs\kanboard\app\Action\TaskDuplicateAnotherProject.php"
https://github.com/LBAK420/Kanboard-Duplicate-a-Task

**Comment by fguillot** [comment]
The automatic action "Duplicate the task to another project" will always duplicate the task to the first column of the destination project. It doesn't offer the possibility to select a different column in the destination project.

**Reply by LBAK420** [reply]
Tnx Sir for ur reply it means in have to register a second action to get the job done?
